spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Smoke test for demo app', function() {
+    
+    browser.waitForAngularEnabled(false);
+    browser.manage().window().maximize();
+    browser.get('http://automationpractice.com/index.php?controller=authentication&back=my-account');
+    
+
+it('Initial build testing', function() {
+    
+    // Enter email in "Email address" field, which is located in "Already registered?" form
+    browser.driver.findElement(by.id("email")).sendKeys("gigedov@getnada.com");
+
+    // Enter password in "Password" field, which is located in "Already registered?" form
+    browser.driver.findElement(by.id("passwd")).sendKeys("testing1");
+
+    // Click on "Sign in" button, which is located in "Already registered?" form
+    browser.driver.findElement(by.id("SubmitLogin")).click();
+    browser.sleep(3000);
+
+    // Enter data in search bar
+    browser.driver.findElement(by.id("search_query_top")).sendKeys("printed summer dress");
+
+    // Click on search icon
+    browser.driver.findElement(by.className("btn btn-default button-search")).click();
+    browser.sleep(3000);
+
+    // Hover over the first showed item
+    browser.actions().mouseMove(element(by.className("product_img_link"))).perform();
```

Comment:
before adding item to chart, can you add following:
- view images
- select item size
- select item quantity